lib/puppet/type/cisco_bgp.rb

```diff
@@ -40,6 +40,7 @@
       cluster_id                             => '55',
       confederation_id                       => '77.6',
       confederation_peers                    => '77.6 88 99.4 200'
+      enforce_first_as			     => true,
```

Comment:
Yeah looks like hard tabs:

```
+      enforce_first_as\t\t\t     => true,
```

Rubocop didn't catch this because it's inside a doc string - not actually Ruby code.